### ISSUE TITLE
Masthead logo isn't retina

### ISSUE THREAD

**Issue by lukecarbis** [wpaustralia/WCBNE18#2]
I'd submit a PR, but I don't have the original source file to recreate the masthead logo @2x (or higher).

**Comment by lukecarbis** [comment]
Potentially, we could restyle this is just plain old text, like the hero logo on the home page?

**Comment by dd32** [comment]
I don't think we have the original artwork, however, it may have been extracted from `WordCamp/Brisbane 2017/Artwork/Website/home-page.psd` in the shared WP BNE google drive.

I think the best option is to re-create it, The element being styled is unfortunately the Site Name which kind of needs to stay as the long-form name for all the other uses of it
